docs/tutorials/02_ui/viz_check_boxes.py

```diff
@@ -0,0 +1,137 @@
+"""
+============================================================
+Figure and Color Control using Check boxes and Radio Buttons
+============================================================
+
+This example shows how to use the CheckBox UI API. We will demonstrate how to
+create a cube, sphere, cone and arrow and control its color and visibility
+using checkboxes.
+
+First, some imports.
+"""
+
+from fury import actor, ui, window, utils
+import numpy as np
+
+###############################################################################
+# We create the corresponding object actors for cube, sphere, cone and arrow.
+
+cube = actor.cube(centers=np.array([[15, 0, 0]]),
+                  colors=np.array([[0, 0, 255]]),
```

Comment:
Thank you for the update! I am still waiting for #283 to be merged. Then you can change this from `255` to `1`.

Reply:
Sure 👍

Reply:
you can go ahead, it has been merged 😄

Reply:
@skoudoro Updated 😄